split.py

```diff
@@ -205,10 +257,22 @@ def main(rom_path, config_path, repo_path, modes, verbose):
 
         if tp not in seg_sizes:
             seg_sizes[tp] = 0
+            seg_split[tp] = 0
+            seg_cached[tp] = 0
         seg_sizes[tp] += segment.rom_length
 
         if segment.should_run():
-            print(f"Splitting {segment.type} {segment.name} at 0x{segment.rom_start:X}")
+            # Check cache
+            cached = segment.cache()
+            if not ignore_cache and cached == cache.get(segment.unique_id()):
+                log.dot(status="skip")
+                seg_cached[tp] += 1
+
```

Comment:
Spacing on this is a bit odd. I'd think the "continue" should be right after the `seg_cached[tp] += ` line and then there should be an empty line separating this if clause